packages/extension/src/newtab/MainFeedPage.tsx

```diff
@@ -97,45 +98,50 @@ export default function MainFeedPage({
   };
 
   return (
-    <MainLayout
-      greeting
-      mainPage
-      isNavItemsButton
-      activePage={activePage}
-      onLogoClick={onLogoClick}
-      showDnd={showDnd}
-      dndActive={isDndActive}
-      onShowDndClick={() => setShowDnd(true)}
-      enableSearch={enableSearch}
-      onNavTabClick={onNavTabClick}
-      screenCentered={false}
-      customBanner={isDndActive && <DndBanner />}
-      additionalButtons={!loadingUser && <CompanionPopupButton />}
-    >
-      <FeedLayout>
-        <MainFeedLayout
-          feedName={feedName}
-          isSearchOn={isSearchOn}
-          searchQuery={searchQuery}
-          onFeedPageChanged={onNavTabClick}
-          searchChildren={
-            <PostsSearch
-              onSubmitQuery={async (query) => setSearchQuery(query)}
-            />
-          }
-          navChildren={!isSearchOn && <ShortcutLinks />}
-          shortcuts={
-            <ShortcutLinks
-              className={classNames(
-                layout === FeedLayoutEnum.Control
-                  ? 'ml-auto'
-                  : 'mt-4 w-fit [@media(width<=680px)]:mx-6',
-              )}
-            />
-          }
-        />
-      </FeedLayout>
-      <DndModal isOpen={showDnd} onRequestClose={() => setShowDnd(false)} />
-    </MainLayout>
+    <>
+      <div className="fixed bottom-0 left-0 z-2 w-full">
```

Comment:
I'm not really sure what this PR is intended to fix/what the issue was.

But setting a main wrapped to fixed is 100% guaranteed to give issues if you ask me.
Can you maybe show me the issue, perhaps we can solve it otherwise? 🙏

Reply:
Well, this is not setting main to fixed. It is adding another element into main that is fixed. Does that make sense?

Originally - the scroll to top button has a fixed position with fixed bottom coordinates. But on iOS in some cases, the browser does not respect that, e.g. the bottom navbar appears to be taller (this happens for example if you display the browser in multitasking on right half of the screen on the iPad). So in cases like that, the bottom navbar can cover the scroll to top button (at least partially). See https://dailydotdev.atlassian.net/browse/MI-100. 

The obvious fix here was to add the scroll to top button to the bottom navbar, hence the coordinates will be always right and the navbar will never cover the button.

Let me know what you think, we can also sync on it together, but you need a physical iOS device for this, since it does not happen on a simulator.

Reply:
You won't have to about mobile on the extension since the extension is only for desktops. We don't normally cover the case.

Reply:
As Lee mentioned if we are only introducing this for extension mobile view I'd rather not do it.
Adds more edge-case then solves them I think?